woo-razorpay.php

```diff
@@ -1923,7 +1923,15 @@ function razorpay_webhook_init()
 {
     $rzpWebhook = new RZP_Webhook();
 
-    $rzpWebhook->process();
+    while(true)
```

Comment:
Didn't understand the below code. Can you elaborate what it does?

Reply:
In case of conflict I am returning 1, in all the other cases null is being returned so I am checking if returned value is not null then I am triggering the process function again and I keep on doing this till 5 minutes have passed after that due to normal execution of the code null will be returned and we will come out of the loop.